src/rmarkdown/index.ts

```diff
@@ -6,6 +6,10 @@ import {
 import { runChunksInTerm } from '../rTerminal';
 import { config } from '../util';
 
```

Comment:
I can't add the comment at the corresponding lines (since it's not actually part of this PR), but I'd definitely prefer to use a `*` import for `vscode` here